Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -827,6 +827,17 @@ protected theorem ciSup_mul_ciSup (g : ι' → Cardinal.{v}) :
     (⨆ i, f i) * (⨆ j, g j) = ⨆ (i) (j), f i * g j := by
   simp_rw [Cardinal.ciSup_mul f, Cardinal.mul_ciSup g]
 
+theorem sum_eq_iSup {ι} {f : ι → Cardinal} (hι : ℵ₀ ≤ #ι) (h : #ι ≤ iSup f) : sum f = iSup f :=
+  le_antisymm (max_eq_right h ▸ mul_eq_max hι (hι.trans h) ▸ sum_le_iSup f) (iSup_le_sum f)
```

Comment:
As much as I enjoy golfing, I think two substs might be a bit too much. It's a bit hard to understand how this proof is operating. Can you rewrite this as a short tactic-mode proof instead?